### PR TITLE
Add Emerald

### PR DESCRIPTION
**Pull request by Bombfuse**
Please, read the [contributing guidelines][contributing] before submitting a PR.

[contributing]: https://github.com/rust-gamedev/rust-gamedev.github.io/blob/source/CONTRIBUTING.md

### CODE REVIEW

File: content/posts/newsletter-020/index.md [hunk 1]
```diff
@@ -66,6 +66,40 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+is a 2D game engine focused on being as portable as possible.
+
+The ultimate goal of Emerald is to be a fully
+featured engine that you can slap onto any
+device with relative ease.
+It's currently able to run on WASM,
+Raspberry Pi, Mac, Windows, and Linux
+thanks to [miniquad][miniquad_git].
+
+Features
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-020/index.md:83]:
```suggestion
Features:
```

File: content/posts/newsletter-020/index.md [hunk 2]
```diff
@@ -66,6 +66,40 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+- Physics via [rapier2d][rapier_2d]
+- ECS via [hecs][hecs_git]
+- Font rendering via [fontdue][fontdue_git]
+
+New Updates:
+
+- Rendering to textures
+
+[WASM game sample][em_wasm_example]
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-020/index.md:93]:
```suggestion
- Rendering to textures
- [WASM game sample][em_wasm_example]
```

**Review by 17cupsofcoffee — COMMENTED** [review]
Some minor formatting suggestions, but looks good otherwise.

File: content/posts/newsletter-020/index.md (hunk 2 above)
Comment (Bombfuse) [reply content/posts/newsletter-020/index.md]: updated

File: content/posts/newsletter-020/index.md (hunk 1 above)
Comment (Bombfuse) [reply content/posts/newsletter-020/index.md]: updated

**Comment by Bombfuse** [comment]
Updated with formatting suggestions! :D

**Review by 17cupsofcoffee — APPROVED** [review]
👍

**Comment by 17cupsofcoffee** [comment]
Thank you for the PR 😄

**Pull request merged**